stream-compositions/services/pom.xml

```diff
@@ -21,4 +21,22 @@
         <module>product-catalog-composition-service</module>
         <module>transaction-composition-service</module>
     </modules>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.projectreactor</groupId>
+            <artifactId>reactor-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
```

Comment:
if we are using the SSDK reactive parent which includes jibs configuration, why do we need this declaration here?